src/j1client.js

```diff
@@ -103,7 +114,30 @@ class JupiterOneClient {
       if (res.errors) {
         throw new Error(`JupiterOne returned error(s) for query: '${j1ql}'`);
       }
-      const { data } = res.data.queryV1;
+
+      const deferredUrl = res.data.queryV1.url;
+      let status = JobStatus.IN_PROGRESS;
+      let statusFile;
+      var sleep = function(ms) {
```

Comment:
Use `let` or `const` (not `var`)

Reply:
Move `sleep` to top-level function in this file (doesn't need to be in the body of this function)